modules/twitch_streaming

```diff
@@ -38,6 +38,7 @@ class Py3status:
 
     def is_streaming(self, i3s_output_list, i3s_config):
         r = requests.get('https://api.twitch.tv/kraken/streams/' + self.stream_name)
+        print(r.json())
```

Comment:
this has returned ;p

`git add -p` is really useful for making sure you only add the bits you need.  It takes a little time to get used to but is great.

likewise `stream_name = "asdfdfgag"` may not be intended.